### PR TITLE
Typecheck util p3

### PR DESCRIPTION
**Pull request by smohsinali**

### CODE REVIEW

File: ConfigSpace/util.py [hunk 1]
```diff
@@ -122,12 +122,12 @@ def get_one_exchange_neighbourhood(configuration, seed):
                 # Activate hyperparameters if their parent node got activated
                 children = configuration.configuration_space.get_children_of(
                     hp_name)
-
+                # todo: debug why mypy is giving error here
                 if len(children) > 0:
-                    to_visit = deque()
+                    to_visit = deque()  # type : ignore
```
Comment (mfeurer) [inline ConfigSpace/util.py]: What happens if you set this to `type: deque`?

Comment (smohsinali) [inline ConfigSpace/util.py]: the problem was space between type and : 
also changed the type to deque

Comment (smohsinali) [inline ConfigSpace/util.py]: there was conflict in hyperparameter.py file...I resolved it so now it should merge

**Comment by smohsinali** [comment]
there was conflict in hyperparameter.py file...I resolved it so now it should merge

**Pull request merged**